# Description: Change location of Comment field to make it less confusing
# Origin: vendor
# Bug-Ubuntu: https://bugs.launchpad.net/bugs/385801
# Forwarded: no
# Author: Chow Loong Jin <hyperair@ubuntu.com>
---

 interfaces/share-dialog.glade.in |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)


diff --git a/interfaces/share-dialog.glade.in b/interfaces/share-dialog.glade.in
index 179ebac..f6d399a 100644
--- a/interfaces/share-dialog.glade.in
+++ b/interfaces/share-dialog.glade.in
@@ -273,8 +273,8 @@
 	    <packing>
 	      <property name="left_attach">0</property>
 	      <property name="right_attach">1</property>
-	      <property name="top_attach">3</property>
-	      <property name="bottom_attach">4</property>
+	      <property name="top_attach">2</property>
+	      <property name="bottom_attach">3</property>
 	      <property name="x_options">fill</property>
 	    </packing>
 	  </child>
@@ -294,8 +294,8 @@
 	    <packing>
 	      <property name="left_attach">1</property>
 	      <property name="right_attach">2</property>
-	      <property name="top_attach">3</property>
-	      <property name="bottom_attach">4</property>
+	      <property name="top_attach">2</property>
+	      <property name="bottom_attach">3</property>
 	      <property name="y_options"></property>
 	    </packing>
 	  </child>
@@ -315,8 +315,8 @@
 	    <packing>
 	      <property name="left_attach">0</property>
 	      <property name="right_attach">2</property>
-	      <property name="top_attach">2</property>
-	      <property name="bottom_attach">3</property>
+	      <property name="top_attach">3</property>
+	      <property name="bottom_attach">4</property>
 	      <property name="x_options">fill</property>
 	      <property name="y_options"></property>
 	    </packing>
